### ISSUE TITLE
A11y_AzureDataStudio_Createnewservergroup_ScreenReader: ORCA screen reader is not announcing "Required" for "Server group name" and "Group description" edit fields.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22691]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #External; #External:AzureDataStudio; #FirstParty:22671; #WCAG3.3.2; #Linux; #ScreenReader; #ORCA; 

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "New Server Group" icon and hit ENTER key.
3. "Add server group" screen gets open. Start screen reader and TAB to "Server group name" and "Group description".
4. Observe that ORCA screen reader is not announcing "Required" for "Server group name" and "Group description" edit fields.

### Actual Result:​
ORCA screen reader is not announcing "Required" for "Server group name" and "Group description" edit fields.

### Expected Result:​
ORCA screen reader should announce "Required" for "Server group name" and "Group description" edit fields as without filling those field we are not able to save the data.

### User Impact:
Screen reader users will not get information about the required field as its not announcing it as required.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

[Video 3-30 at 12.37 (2).webm](https://user-images.githubusercontent.com/90901519/228807508-97a893ac-abd2-4989-a800-c19a39a663d9.webm)
